README.md

```diff
@@ -111,9 +111,11 @@ def send_to_slack(sender, instance, source_object, translated_object, **kwargs):
     )
 
     edit_url = "https://www.mysite.com" + get_edit_url(translated_object)
+    locale = translated_object.locale.get_display_name()
+    message = f"'{source_object.title}' has new translations for the '{locale}' locale. See the updated page at: {edit_url}."
```

Comment:
question: should this be translateable?

Reply:
I think for example code in a README it is actually useful to keep it as minimal as possible but I don't have incredibly strong feeling on it

Reply:
Done in b4d9477
The examples have got pretty verbose at this point anyway.